cogs/meme.py

```diff
@@ -155,7 +157,16 @@ async def frolics(self, ctx):
         """test"""
         await ctx.send("https://www.youtube.com/watch?v=VmarNEsjpDI")
 
-    @commands.command(hidden=True, aliases=["yotld", "yold", "yoltd", "yearoflinuxondesktop", "yearoflinuxonthedesktop"])
+    @commands.command(
```

Comment:
Is this the result of black pass or did you use another formatter?

Reply:
Black, yeah.